lib/StripeResource.js

```diff
@@ -261,6 +271,11 @@ StripeResource.prototype = {
     // But never sleep less than the base sleep seconds.
     sleepSeconds = Math.max(initialNetworkRetryDelay, sleepSeconds);
 
+    // And never sleep less than the time the API asks us to wait, assuming it's a reasonable ask.
+    if (Number.isInteger(retryAfter) && retryAfter <= 60) {
```

Comment:
Can we constant-ize `60` here to make it more literate? Maybe `MAX_RETRY_AFTER_WAIT`?